source/configuration/write.txt

```diff
@@ -96,25 +96,15 @@ The following options for writing to MongoDB are available:
        | **Default:** ``_id``
 
    * - ``writeConcern.w``
-     - | Specifies ``w``, a write concern option to acknowledge the level to
-         which the change propagated in the MongoDB replica set. You can
-         specify one of the following values:
-
-       - ``MAJORITY``
-       - ``W1``, ``W2``, or ``W3``
-       - ``ACKNOWLEDGED``
-       - ``UNACKNOWLEDGED``
-
+     - | Specifies ``w``, a write-concern option to acknowledge the level to
+         which the change propagated in the MongoDB replica set. For a list
+         of allowed values for this option, see :manual:`WriteConcern
+         </reference/write-concern/#w-option>` in the MongoDB manual.
        |
-       | For more information on ``w`` values, see the MongoDB server guide on
-         the :manual:`WriteConcern w option
-         </reference/write-concern/#w-option>`.
-
-       |
-       | **Default:** ``ACKNOWLEDGED``
+       | **Default:** ``1`` (acknowledged)
```

Comment:
[suggestion]

Would it make sense to remove "(acknowledged)"? Without the context of what this value means, I find it distracting rather than informative. Since the names are no longer a valid option, maybe just have a value of "1" and let folks go to the linked doc for more details?